onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -412,50 +438,54 @@ def testLabelEncoder(self):
         self.assertEqual(output_type, "tensor(int64)")
         output_shape = sess.get_outputs()[0].shape
         self.assertEqual(output_shape, [1, 1])
-        
+
         # Array
         x = np.array([['4']])
         res = sess.run([output_name], {input_name: x})
         output_expected = np.array([[3]], dtype=np.int64)
-        np.testing.assert_allclose(output_expected, res[0], rtol=1e-05, atol=1e-08)
+        np.testing.assert_allclose(
+            output_expected, res[0], rtol=1e-05, atol=1e-08)
 
         # Python type
         x = np.array(['4'])
```

Comment:
isn't the issue a bug with the test itself? input should be [1,1] but we're passing [1] here and expecting [1] output